base.scss

```diff
@@ -0,0 +1,68 @@
+h1 {
```

Comment:
Generally it is better to be specific with element selectors and avoid universal selectors - prevents from styling elements that you didn't want to style, and a bit better performance